src/org/sosy_lab/java_smt/api/StringFormulaManager.java

```diff
@@ -0,0 +1,34 @@
+// This file is part of JavaSMT,
+// an API wrapper for a collection of SMT solvers:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2021 Alejandro Serrano Mena
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.sosy_lab.java_smt.api;
+
+/** Manager for dealing with string formulas. */
+public interface StringFormulaManager {
+
+  /**
+   * Returns a {@link StringFormula} representing the given constant.
+   *
+   * @param value the boolean value the returned <code>Formula</code> should represent
+   * @return a Formula representing the given value
+   */
+  StringFormula makeString(String value);
+
+  /**
+   * Creates a variable with exactly the given name.
+   *
+   * <p>Please make sure that the given name is valid in SMT-LIB2. Take a look at {@link
+   * FormulaManager#isValidName} for further information.
+   *
+   * <p>This method does not quote or unquote the given name, but uses the plain name "AS IS".
+   * {@link Formula#toString} can return a different String than the given one.
+   */
+  StringFormula makeVariable(String pVar);
+
+  BooleanFormula equal(StringFormula str1, StringFormula str2);
+}
```

Comment:
Is it useful (and possible) to already include more String-based methods here?
According to [SMTLIB2 String theory](http://smtlib.cs.uiowa.edu/theories-UnicodeStrings.shtml) the following methods might be possible: `concat`, `length`, and `compare` (maybe represented as `lessThan`/`greaterThan`).
Regular expressions might still be out of scope, but remain as future work (not in this PR).